client/pages/search-results.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Result from '../components/result';
+import Map from '../components/map';
+
+export default class SearchResults extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      results: [],
+      center: null
+    };
+  }
+
+  componentDidMount() {
+    const url = new URL('localhost:3000/search-results');
+    for (const key in this.props) {
+      if (this.props) url.searchParams.append(key, this.props[key]);
+    }
+    fetch(`/search-results${url.search}`)
```

Comment:
We can just fetch the URL we created

```js
fetch(url)
```

https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters